app/views/spree/products/_filters_desktop.html.erb

```diff
@@ -0,0 +1,62 @@
+          <button class="card-header <%= 'collapsed' if collapsed %> px-1 plp-filters-card-header" id="filters<%= option_type_name %>" data-toggle="collapse" data-target="#collapseFilter<%= option_type_name %>" aria-expanded="<%= !collapsed %>" aria-controls="collapseFilter<%= option_type_name %>" role="heading" aria-level="2">
+            <%= option_type.presentation %>
+            <%= icon(name: 'plus',
+                    classes: 'mt-1 float-right arrow plp-filters-card-header-arrow--plus',
+                    width: 16,
+                    height: 16) %>
+            <%= icon(name: 'minus',
+                    classes: 'mt-1 float-right arrow plp-filters-card-header-arrow--minus',
+                    width: 16,
+                    height: 16) %>
+          </button>
+          <div id="collapseFilter<%= option_type_name %>" class="collapse <%= 'show' unless collapsed %>" aria-labelledby="filters<%= option_type_name %>">
+            <div class="card-body px-0 pt-1">
+              <%= render 'spree/shared/option_values',
+                params_ot_downcase_name: params[ot_downcase_name],
+                option_type: option_type,
+                permitted_params: permitted_params,
+                ot_downcase_name: ot_downcase_name
+              %>
+            </div>
+          </div>
+        </div>
+      <% end %>
+      <% price_collapsed = params[:price].blank? %>
+      <div class="w-100 card plp-filters-card">
+        <button class="card-header text-capitalize <%= 'collapsed' if price_collapsed %> px-1 plp-filters-card-header" id="filtersPrice" data-toggle="collapse" data-target="#collapseFilterPrice" aria-expanded="<% !price_collapsed %>" aria-controls="collapseFilterPrice" role="heading" aria-level="2">
+          <%= Spree.t('plp.price') %>
+          <%= icon(name: 'plus',
+                  classes: 'mt-1 float-right arrow plp-filters-card-header-arrow--plus',
+                  width: 16,
+                  height: 16) %>
+          <%= icon(name: 'minus',
+                  classes: 'mt-1 float-right arrow plp-filters-card-header-arrow--minus',
+                  width: 16,
+                  height: 16) %>
+        </button>
+        <div id="collapseFilterPrice" class="collapse <%= 'show' unless price_collapsed %>" aria-labelledby="filtersPrice">
+          <div class="card-body px-0 pt-1">
+            <% price_filter_values.each do |name| %>
+              <% price_param = params[:price] == name ? '' : name %>
+              <%= link_to permitted_params.merge({ price: price_param }), class: "d-block" do %>
+                <div class="d-inline-block py-1 px-2 m-1 text-uppercase plp-filters-card-item <%= 'plp-filters-card-item--selected' if params[:price] == name %>">
+                  <%= name %>
+                </div>
+              <% end %>
+            <% end %>
+          </div>
+        </div>
+      </div>
+      <%= render 'spree/products/vendors_desktop_filter', permitted_params: permitted_params %>
```

Comment:
I think it could be like that as a temporary measure, but I would suggest removing this file from this repo and making a loop in spree/spree repository's `_filters_desktop.html.erb` so that the filters are easy to extend and we don't have to update this repository every time there's a change in filters views in spree/spree.